spec/models/pglogical_subscription_spec.rb

```diff
@@ -241,6 +273,9 @@
 
       expect { sub.save! }.to raise_error(RuntimeError, "Some Error")
     end
+
+    it "raises when the schemas don't match" do
```

Comment:
Wow I just didn't write content for this :sweat_smile: 

I think this test was supposed to be removed and replaced with the other "raises when *" tests I added